astropy/coordinates/tests/test_stokes_coord.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_equal
+
+from astropy.coordinates.stokes_coord import (
+    StokesCoord,
+    StokesSymbol,
+    custom_stokes_symbol_mapping,
+)
+from astropy.utils import unbroadcast
+
+
+def test_scalar():
+    sk = StokesCoord(2)
+    assert repr(sk) == "<StokesCoord 'Q'>"
+    assert sk.value == 2.0
+    assert sk.symbol == "Q"
+
+
+def test_vector():
+    # This also checks that floats are rounded when converting
+    # to strings
+    values = [1.2, 1.8, 2.0, 2.2, 2.8]
+    sk = StokesCoord(values)
+    assert_equal(sk.value, values)
+    assert_equal(sk.symbol, np.array(["I", "Q", "Q", "Q", "U"]))
+    assert repr(sk) == "<StokesCoord ['I', 'Q', 'Q', 'Q', 'U']>"
```

Comment:
So, I *think* it would be nicer as ``StokesCoord(['I',` 'Q', 'Q', 'Q', 'U'])`, i.e., more like `ndarray`:
```
In [6]: np.arange(10.)
Out[6]: array([0., 1., 2., 3., 4., 5., 6., 7., 8., 9.])
```